Assets/Scripts/PlayerInventory.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEngine;
+
+public class PlayerInventory : MonoBehaviour
+{
+    public LayerMask ignoreLayers = 1 << 4;
```

Comment:
Hardcoding layer references is not portable and is discouraged. Suggest just setting "Default" (1ignoreLayers = 1) as the fallback and rely on prefabs to scenes to change this value via the inspector.